server/db/stargazers.js

```diff
@@ -0,0 +1,21 @@
+// Schema of stargazers in Github
+const mongoose = require('./mongoose');
+
+const Schema = mongoose.Schema;
+
+const stargazersSchema = new Schema({
+    githubId: Number,
+    login: String,
+    name: String,
+    html_url: String,
+    location: String,
+    bio: String,
+    public_repos: Number,
+    public_gists: Number,
+    followers: Number,
+}, { timestamps: { updatedAt: 'recordUpdated_at', createdAt: 'recordCreated_at' } });
```

Comment:
I think you can write this object in multiple lines. A one liner is harder to read + the diff in the future might not be clear.